lib/remote/configpackageshandler.cpp

```diff
@@ -20,23 +21,25 @@ bool ConfigPackagesHandler::HandleRequest(
 {
 	namespace http = boost::beast::http;
 
-	auto url = request.Url();
-	auto user = request.User();
-	auto params = request.Params();
-
-	if (url->GetPath().size() > 4)
-		return false;
-
-	if (request.method() == http::verb::get)
-		HandleGet(request, response);
-	else if (request.method() == http::verb::post)
-		HandlePost(request, response);
-	else if (request.method() == http::verb::delete_)
-		HandleDelete(request, response);
-	else
-		return false;
-
-	return true;
+	return QueueAsioFutureCallback([&]() {
+		auto url = request.Url();
+		auto user = request.User();
+		auto params = request.Params();
+
+		if (url->GetPath().size() > 4)
+			return false;
+
+		if (request.method() == http::verb::get)
+			HandleGet(request, response);
+		else if (request.method() == http::verb::post)
+			HandlePost(request, response);
+		else if (request.method() == http::verb::delete_)
+			HandleDelete(request, response);
+		else
+			return false;
```

Comment:
What about wrapping at the Handle*() level? I mean, what a waste to deploy a callback which will almost immediately return false..